netlify/netlify.toml

```diff
@@ -1,14 +1,6 @@
 [build]
-command = "npm run build"
+command = "react-router build"
 publish = "build/client"
 
 [dev]
-command = "npm run dev"
-framework = "vite"
-
-# Set immutable caching for static files, because they have fingerprinted filenames
-
-[[headers]]
-for = "/assets/*"
-[headers.values]
-"Cache-Control" = "public, max-age=31560000, immutable"
```

Comment:
This isn't needed, since these assets are in the publish dir (`build/client`) and uploaded to the CDN at deploy time and served statically